src/index.html

```diff
@@ -1,16 +1,16 @@
-<!DOCTYPE html>
-<html>
-	<head>
-		<meta charset="UTF-8">
-
-		<title>Solitaire</title>
-	</head>
-
-	<body>
-		<div class="app-container js-app-container">
-			<canvas class="app-canvas js-app-canvas">
-				Your browser does not support canvas elements.
-			</canvas>
-		</div>
-	</body>
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit="no">
+    <title>Solitaire</title>
+  </head>
+  <body>
+    <div class="app-container js-app-container">
+      <canvas class="app-canvas js-app-canvas">
+        Sorry, your browser doesn’t support canvas elements.
```

Comment:
Missing support is not the fault of the page and the page should not apologize.
This line should be reverted.
:shrug: :smile: